src/index.js

```diff
@@ -12,7 +12,13 @@ const logger = require('./logger');
 // without starting the server itself
 const app = server();
 
-app.listen(config.get('app:port'), () => {
+const attachedApplication = app.listen(config.get('app:port'), () => {
   logger.info(`Listening on port ${config.get('app:port')}!`);
   metrics.metricsServer();
 });
+
+if (config.get('settings:keep_alive_timeout') !== '') {
```

Comment:
@codegaze Maybe `if (config.get('settings:keep_alive_timeout') > 0) {` is better because if you set the config or environment variable to 'abcd' it will resolve to setting keepAliveTimeout to `NaN`

Reply:
@nbasili I wanted to avoid 0 because it's a a valid value. Will change it either way.